apps/docs/src/guide/cookbook/combining-utxos.md

```diff
@@ -9,3 +9,9 @@ One way to avoid these errors is to combine your UTXOs. This can be done by perf
 > **Note:** You will not be able to have a single UTXO for the base asset after combining, as one output will be for the transfer, and you will have another for the fees.
 
 <<< @./snippets/combining-utxos.ts#combining-utxos{ts:line-numbers}
+
+## Max Outputs
+
+It's also important to note that depending on the chain configuration, you may be limited on the number of inputs and/or outputs you can have in a transaction. These amounts can be queried via the [TxParameters](https://docs.fuel.network/docs/graphql/reference/objects/#txparameters) GraphQL query
+
+<< @./snippets/max-outputs.ts#max-outputs{ts:line-numbers}
```

Comment:
```suggestion
<<< @./snippets/max-outputs.ts#max-outputs{ts:line-numbers}
```

The missing `<` results in the snippet not rendering properly:

<img width="391" alt="Screenshot 2024-12-04 at 7 57 24 AM" src="https://github.com/user-attachments/assets/22eac530-fc0e-4c4c-95fd-42bca25908ca">